amethyst_tiles/src/map.rs

```diff
@@ -42,12 +43,16 @@ pub trait Map {
     /// Convert a tile coordinate `Point3<u32>` to an amethyst world-coordinate space coordinate `Point3<f32>`
     /// This performs an inverse matrix transformation of the world coordinate, scaling and translating using this
     /// maps `origin` and `tile_dimensions` respectively.
-    fn to_world(&self, coord: &Point3<u32>) -> Vector3<f32>;
+    fn to_world(&self, coord: &Point3<u32>, map_transform: Option<&Transform>) -> Vector3<f32>;
```

Comment:
We should enhance the documentation to indicate what this new parameter is used for and how it alters the behavior.

Reply:
I have done that now. Thanks.